src/index.html

```diff
@@ -11,6 +11,41 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="search-bar-big"
+      data-qa="big"
+      action="find"
+      method="get"
+    >
+      <img
+        src="/images/Search.svg"
+        alt="search icon"
+        class="search-bar-big__icon"
+      >
+      <input
```

Comment:
add one empty line between same level elements
```suggestion
      >
      
      <input
```